modules/cluster-logging-elasticsearch-retention.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-elasticsearch.adoc
+
+[id="cluster-logging-elasticsearch-retention_{context}"]
+= Configuring Elasticsearch retention policy
+
+
+Each Elasticsearch index adds load and overhead to the Elasticsearch cluster. You can use the Elasticsearch rollover index API to automatically rollover indices when they reach a specified age. When an index reaches the designated age, Elasticsearch rolls over the index, moving the current index and creating a new index. After seven days, the old index is automatically deleted. Rolling over and deleting indices optimizes the indices and helps Elasticsearch performance. 
+
+You specify how long Elasticsearch stores indices using a separate _retention policy_ for each of the three log sources, infrastucture logs, application logs, and audit logs, using the Cluster Logging Custom Resource (CR) to set the number of days after which the indices should be rolled over. The Cluster Logging CR writes the full policy to the Elasticsearch Custom Resource (CR).
+
+For example, the following Elasticsearch CR has a retention policy to roll over active indices for the infrastructure logs every three days and the old indices are deleted seven days after rollover. {product-title} checks every 15 minutes to determine if the infrastructure indices need to be rolled over.
+
+----
+apiVersion: "logging.openshift.io/v1"
+kind: "Elasticsearch"
+metadata:
+  name: "elasticsearch"
+spec:
+  indexManagement:
+    policies: <1>
+    - name: infra-policy
+      pollInterval: 15m
+      phases:
+        hot:
+          actions:
+            rollover:
+              maxAge: 3d
+        delete:
+          minAge: 7d
+    mappings: <2>
+    - name:  node.infra 
+      policyRef: infra-policy
+      aliases:
+      - infra  
+...
+----
+<1> For each log source, the retention policy indicates when to delete and rollover logs for that source. The policy includes: 
+* When to rollover the indices, determine by `maxAge`. This setting comes from the Cluster Logging CR.
+* When to check if rollover should be performed, determined by `pollInterval`. This setting is the default and cannot be changed.
+* When the rolled-over indicies will be deleted after the rollover, determined by `minAge`. This setting is the default and cannot be changed.
```

Comment:
The `minAge` in the elasticsearch CR equals to the `maxAge` in the clusterlogging CR.